tests/OpenChallenge.tex

```diff
@@ -28,19 +31,15 @@ \subsection{Jury evaluation}
   \item \textbf{Jury of team leaders:} All teams have to provide \emph{one} person 
   (preferably the team-leader) to follow and evaluate the entire Open Challenge.
   \item \textbf{Evaluation:} Both the demonstration of the robot(s), and the answers of the team in the interview part are evaluated.\\ 
-  For each of the following \emph{evaluation criteria}, a maximum of \scoring{10 points} is given per jury member:
+  For each of the following \emph{evaluation criteria}, each jury member submits a score from $0-100$:
   \begin{enumerate}
-  \item Overall demonstration
```

Comment:
I wouldn't remove this. There can be innovative and successful presentations carried out in a very boring way.